README.md

```diff
@@ -19,6 +19,23 @@ If you need to install docker and/or docker-compose, you can use the script in t
 $ ./docker_install_ubuntu.sh
 ```
 
+## Gotcha
+This repo has a directory called 'volttron', which contains the volttron codebase. In other words, this repo contains another repo in a subfolder. 
+When you initially clone this repo, the 'volttron' directory is empty. This directory contains the volttron codebase used to create the volttron platform. Before creating the container, you must pull in volttron from the official volttron repo using the following git command:
+
+```bash
+# Ensure that you are in the `volttron directory` 
```

Comment:
> Instead of a Gotcha please add that to a building image locally section. For the most part what we want is to not have the user have to do that and just use the image that is out on dockerhub.
> 
> Including a building/building the latest code branch section would emphasis this.

Thanks for the clarification. I've been building and testing locally for so long that I didn't realize this step is not necessary to run docker-compose. I just emptied that folder and ran `docker-compose up` and it runs fine. I already created a local development section, so I'll move this to there.